0000-relationship-between-protocols/0000-relationship-between-protocols.md

```diff
@@ -0,0 +1,88 @@
+
+## Scope
+The scope of this document is the explication of the relationship between the Interledger protocols, and the details for it are out of scope. Some references are given for it though. i.e. This document draws a whole, big picture of ILP.
+
+## Perspective
+First of all, the diagram below shows the entire relationship between the protocols.
+
+![Perspective](images/perspective.svg)
+
+### Elements and Brief Explanations
+
+- Connections
+  - (A) BTP over WebSocket
+    - **Node** to **Node** connection
+    - In order to exchange payments, configuration and routing information, a node MUST have a WebSocket connection to another node.
+  - (B) Ledger specific connection
+    - **Node** to **Ledger** connection
+    - A node needs the means to settle its payments. Therefore the node has a connection to a ledger. The type of connection varies depending on the ledger.
+  - (C) SPSP over HTTPS
+    - **Application** to **SPSP Server** connection
+    - To determine payment information such as a shared secret, a destination address and so forth, an application connects to SPSP server over HTTPS.
+- Protocols and related specifications
+  - (1) BTP
+    - [Bilateral Transfer Protocol 2.0 (BTP/2.0)](../0023-bilateral-transfer-protocol/0023-bilateral-transfer-protocol.md)
+    - BTP is used for transferring ILP packets and messages that are used for settlements and so on between two nodes.
+  - (2) ILP
+    - [Interledger Protocol V4 (ILPv4)](../0027-interledger-protocol-4/0027-interledger-protocol-4.md)
+    - ILP is used for transferring payment, node configuration, and routing information.
+  - (3) STREAM
+    - [STREAM - A Multiplexed Money and Data Transport for ILP](../0029-stream/0029-stream.md)
+    - STREAM is used for transferring money and data bidirectionally from applications.
+  - (4) Application
+    - The extensible data area of STREAM packets is used arbitrarily by applications.
+  - (5) DCP
+    - [Interledger Dynamic Configuration Protocol (ILDCP) v1](../0031-dynamic-configuration-protocol/0031-dynamic-configuration-protocol.md)
+    - DCP is used in order to exchange node information such as an ILP address.
+  - (6) RBP
+    - Route Broadcasting Protocol
+    - RBP is used for transferring routing information to build routing tables.
+  - (7) SPSP
+    - [The Simple Payment Setup Protocol (SPSP)](../0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md)
+    - SPSP is used to determine payment information such as a shared secret, a destination address and so forth.
+  - (8) ASN.1
```

Comment:
ASN.1 and OER aren't protocols so I would put them in a different section (like Encoding) or as a note after this list